addons/web/static/src/legacy/scss/bootstrap_overridden_frontend.scss

```diff
@@ -2,6 +2,11 @@
 // See https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast
 $min-contrast-ratio: 3 !default;
 
+
+// Body
+
+$body-bg: o-color('white') !default;
```

Comment:
`$form-range-thumb-bg` variable definition needs it https://github.com/odoo/odoo/pull/98558/files/37c329b05390901594fcfebe45bd0da6bb839ecb#diff-152d80d5aee4f0291bf47b9b7be1f4049fc83df2d6e043c3c022001d29063a7eR33

Reply:
@stefanorigano Ok, my bad, should have seen it. Although I guess since I ask you to not use o-color in here, it's a problem. In fact scratch all my comment related to color: nothing in web_editor, everything as you did (in web). But instead of o-color, use the related BS variable (which you'll have to define back in the file when you use it:
```scss
$white: white !default;
[...]
$body-bg: white !default;
[...]
$form-range-thumb-bg: $body-bg !default;
```
(same for the other color you use). That way the weirdness of moving to web_editor is gone and it's technically correct (just ugly you have to repeat the color but well...).